guides/source/active_record_validations.md

```diff
@@ -792,7 +797,8 @@ for each validation helper. The `:message` option accepts a `String` or `Proc`.
 
 A `String` `:message` value can optionally contain any/all of `%{value}`,
 `%{attribute}`, and `%{model}` which will be dynamically replaced when
-validation fails.
+validation fails. This replacement is done using the I18n gem, and the
```

Comment:
Well I guess that's just Ruby, that's not up to the I18n gem:

~~~ruby
>> "%{ foo }" % { foo: "bar" }
KeyError: key{ foo } not found
from (pry):1:in `%`
>> "%{foo}" % { foo: "bar" }
=> "bar"
~~~

Maybe you can just say:

> The placeholders must match exactly, no spaces are allowed.